client/signup/config/steps-pure.js

```diff
@@ -721,6 +721,12 @@ export function generateSteps( {
 				cartItem: PLAN_ECOMMERCE_MONTHLY,
 			},
 		},
+
+		design: {
```

Comment:
Note to self: this is how an onboarding step is defined. This step can then be used in onboarding flows ([example](https://github.com/Automattic/wp-calypso/pull/51501/files#r603335613))
The `stepName` will define the URL that the user will see in their browser when on this step